src/controller/java/AndroidCallbacks.h

```diff
@@ -138,6 +139,20 @@ struct InvokeCallback : public app::CommandSender::Callback
     JniGlobalReference mWrapperCallbackRef;
 };
 
+struct ExtenedableInvokeCallback : public app::CommandSender::ExtendableCallback
```

Comment:
seems like an extra `e` compared with the other other one:

`ExtendableCallback`
`ExtenedableInvokeCallback `